releases/cloud/point-2023-3-release-note.md

```diff
@@ -0,0 +1,63 @@
+---
+description: >-
+  This article lists new features, improvements, and bug fixes in Syskit Point Cloud
+  version 2023.3.
+---
+
+# Syskit Point 2023.3
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2023.3.0
+
+**Build number:** XXX
+
+**Release date:** July 12, 2023
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside of working hours to ensure minimal interference with your day-to-day business.
+
+## Features
+
+* **Introducing the Policy Automation feature!** Syskit Point Admins can now define rules, which Syskit Point will follow, and, automatically apply defined governance policies for workspaces that meet the conditions configured in the said rules.
+You can fine-tune rules to apply governance policies to workspaces based on:
```

Comment:
maybe we could add somewhere in the text that support for custom metadata will come later on since this is something that customers are interested in. This way, they have the answer to their question immediately and there will be no potential disappointment or support inquiries :)